inference/models/yolov8/yolov8_object_detection.py

```diff
@@ -38,10 +39,15 @@ def predict(self, img_in: np.ndarray, **kwargs) -> Tuple[np.ndarray]:
         Returns:
             Tuple[np.ndarray]: NumPy array representing the predictions, including boxes, confidence scores, and class confidence scores.
         """
+        t = perf_counter()
         predictions = self.onnx_session.run(None, {self.input_name: img_in})[0]
+        print(f"Took {perf_counter() - t} seconds to onnx infer")
```

Comment:
We probably want to use a logger that won't always print this

Reply:
yeah, this was debug stuff -- removed